.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: dart
 sudo: false
 dart:
-  - dev
+  - beta
```

Comment:
Can/should we just use dart stable, instead of beta?

Reply:
It's good to be aware of any forthcoming breaking changes, so I'd leave both (it didn't cause any issues in the past).